examples/ex5p.cpp

```diff
@@ -317,6 +317,8 @@ int main(int argc, char *argv[])
    VisItDataCollection visit_dc("Example5-Parallel", pmesh);
    visit_dc.RegisterField("velocity", u);
    visit_dc.RegisterField("pressure", p);
+   // To save the mesh using MFEM's parallel mesh format:
+   // visit_dc.SetFormat(DataCollection::PARALLEL_FORMAT);
```

Comment:
Perhaps this line, and the corresponding line in ex9p, could be controlled by a command line option.

Reply:
I added a command line option in example 5p and its PETSc version.

In ex9p, there are already a number of options for the output, so I think it's better not to add another one. I'll just leave the comment as is.